app/base/handlers.js

```diff
@@ -83,19 +83,19 @@ exports.auth_callback = [
 exports.healthz = (req, res) => res.sendStatus(200);
 
 
-function ip_whitelisted(req, res, next) {
-  if (config.ip_whitelist.length) {
-    const ip = ip_from_request(req);
+function ipWhitelisted(req, res, next) {
+  if (config.ipWhitelist.length) {
+    const ip = ipFromRequest(req);
 
-    if (!config.ip_whitelist.find(range => ip.match(ipaddr.parseCIDR(range)))) {
+    if (!config.ipWhitelist.find(range => ip.match(ipaddr.parseCIDR(range)))) {
       throw new Error(`IP ${ip} is not whitelisted`);
     }
   }
   next();
 }
 
 
-function ip_from_request(req) {
+function ipFromRequest(req) {
   const ips = req.header('X-Forwarded-For') || req.connection.remoteAddress;
```

Comment:
```suggestion
  const ips =req.connection.remoteAddress;
```
can you delete the `x-forwarded-for` thing... I was discussing this with @davidread today and we think it should go

Reply:
Is it safe to remove? This is added by the proxies (and potentially by an attacker :|) but would the IP checking work without it?

Reply:
Yes it's safe to remove, we whitelist the IP of the dom1 and quantum proxies rather than their end users so it doesn't matter. It was added because elb used to add it but nlb is transparent so it's fine without

Reply:
OK, I'll remove 👍